thoth/storages/ceph_cache.py

```diff
@@ -46,11 +46,11 @@ def retrieve_document_ttl(self, document_id: str) -> float:
 
         # Uses UTC time to be environment agnostic (no timezone)
         time_lived = (
-            datetime.now()
+            datetime.now(timezone.utc)
             - self.ceph.retrieve_document_attr(object_key=document_id, attr="LastModified").replace(tzinfo=timezone.utc)
         ).total_seconds()
 
         if time_lived <= 14400.0:
-            return time_lived
+            return 14400.0 - time_lived
 
         return 0.0
```

Comment:
```suggestion
        return timedelta()
```
Only if we switch to a timedelta return.